tests/acceptance/04_examples/check_syntax.cf

```diff
@@ -0,0 +1,95 @@
+#######################################################
+#
+# Check the example syntax
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+      "basedir" string => "$(G.cwd)/../../examples";
+      "all_examples" slist => findfiles("$(basedir)/*.cf");
+
+  reports:
+    DEBUG_LOTS::
+      "$(this.bundle): will consider example $(all_examples)";
+}
+
+#######################################################
+
+bundle agent test
+{
+  meta:
+      # It's unrealistic to keep this test perfectly stable on all non-Linux platforms.
+      # It would require examples that quickly get very cluttered.
+      "test_skip_unsupported" string => "!linux";
+
+  vars:
+      "examples" slist => { @(init.all_examples) };
+      "canon[$(examples)]" string => canonify($(examples));
+
+  methods:
+      "$(examples)" usebundle => test_example($(examples));
+
+  reports:
+    DEBUG::
+      "$(this.bundle): found example with output $(examples)"
+        if => "has_output_block_$(canon[$(examples)])";
+}
+
+bundle agent test_example(file)
+{
+  vars:
+      "cfile" string => canonify($(file));
+      "checker"
+        # -c option is omitted, because it enforces bundlesequence / main
+        string => "$(sys.cf_promises) --workdir=$(G.testdir)";
+
+  classes:
+      "failure_$(cfile)"
+        not => returnszero("$(checker) $(file)", "noshell"),
+        scope => "namespace";
+
+  reports:
+    DEBUG::
+    "Ran checker '$(checker) $(file)'"
+      handle => "printed_command_$(cfile)";
+    "Syntax error in example: '$(file)'"
+      depends_on => { "printed_command_$(cfile)" },
+      if => "failure_$(cfile)",
+      handle => "reported_error_$(cfile)";
+
+  methods:
+    "verbose_$(cfile)"
+      usebundle => verbose_output("$(checker) $(file)"),
```

Comment:
So, you run the checker again, with verbose output so that you can display the specific error?

Reply:
No, not run with verbose output, just run as a commands promise.

Reply:
Note that many things are copied from the check_outputs test. I changed the name here to `run_with_output`.